packages/engine/src/shared-context.ts

```diff
@@ -323,3 +323,46 @@ export type HasContextualSQLModuleExecutors = {
    */
   readOnlyContextualSQLModuleExecutors: ContextualSQLModuleExecutors;
 };
+
+/**
+ * Named URLs to databases.
+ */
+type Databases = {
+  [index: string]: string;
+};
+
+/**
+ * The all important root configuration. This tells EmbraceSQL which databases to manage.
+ */
+export type Configuration = {
+  /**
+   * The root directory used to start this config.
+   */
+  embraceSQLRoot: string;
+  /**
+   * URL identifying Kafka topic or REST endpoing to post commands.
+   */
+  twinCommandsTo?: string;
+  /**
+   * All available databases.
+   */
+  databases?: Databases;
+  /**
+   * Flag for embedded mode. This will be force set by the embedded cli.
+   */
+  embedded: boolean;
```

Comment:
Remove this — generate embedded always — it is the core engine for the plain server.